sms/v1/openapi.yaml

```diff
@@ -436,27 +435,29 @@ paths:
           content:
             application/json:
               schema:
-                type: array
-                items:
-                  $ref: "#/components/schemas/PhoneProvisioningArrayResponse"
-            "*/*":
-              schema:
-                type: array
-                items:
-                  $ref: "#/components/schemas/PhoneProvisioningArrayResponse"
```

Comment:
The `*/*` responses seem to be extra here. Am I right?

Reply:
correct